docs/style.css

```diff
@@ -0,0 +1,565 @@
+}
+
+.container {
+    box-sizing: border-box;
+    width: 1360px;
+    margin-left: 40px;
+    margin-right: 40px;
+}
+
+.footer.container {
+    display: flex;
+    justify-content: space-around;
+    gap: 100px;
+    padding: 100px;
+    text-align: left;
+}
+
+.offer {
+    width: 530px;
+    height: 250px;
+}
+
+.footer-Text {
+    margin-top: -60px;
+    max-height: 150px;
+    margin-left: -60px;
+}
+
+.contacts-info {
+    color: var(--light);
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 150%;
+    gap: 40px;
+    margin-left: -400px;
+}
+
+.about.container {
+    text-align: justify;
+}
+
+.image.Frappuccino, .image.Macchiato, .image.Ice-coffee {
+    width: 480px;
+    height: 480px;
+    object-fit: contain;
+}
+
+.image1, .image2, .image3 {
+    display: grid;
+    grid-auto-flow: row;
+    gap: 20;
+    align-items: center;
+}
+
+.social {
+    display: inline-grid;
+    grid-auto-flow: row;
+    max-height: 100px;
+    max-width: 204px;
+    max-width: fit-content;
+    gap: 12px;
+}
+
+.row-slider {
+    display: grid;
+    grid-auto-flow: column;
+    max-width: 1152px;
+    overflow:scroll;
+    grid-template-columns: 1fr;
+    align-self: center;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+.slider.container {
+    display: flex;
+    flex-direction: row;
+    max-width: 1360px;
+    width: 100%;
+    height: 640px;
+    align-items: center;
+    justify-content:space-between ;
+    gap: 40px;
+}
+
+.slider-button {
+    border: none;
+    width: 59px;
+    height: 59px;
+    background-color: var(--light);
+}
+
+.slider-button:hover {
+    border-radius: 50%;
+}
+
+.right-btn svg:hover  {
+    stroke: var(--light);
+    fill: var(--container);
+}
+
+.right-btn svg {
+    fill: var(--light);
+    stroke: #665F55;
+}
+
+.left-btn svg:hover  {
+    stroke: var(--light);
+    fill: var(--container);
+}
+
+.left-btn svg {
+    fill: var(--light);
+    stroke: #665F55;
+}
+
+.appstore-btn, .playstore-btn{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid var(--container);
+    gap: 8px;
+    border-radius: 100px;
+    width: 200px;
+    height: 64px;
+    padding: 12px 20px;
+    background: var(--light);
+}
+
+.playstore-btn svg:hover {
+    fill: var(--light);
+}
+
+.appstore-btn svg:hover {
+    fill: var(--light);
+}
+
+.apple-store svg:hover {
+    stroke: var(--light);
+}
+
+.playstore-btn svg {
+    /* stroke: var(--dark); */
+    fill: var(--dark);
+}
+
+.appstore-btn svg {
+    fill: var(--dark);
+    stroke: var(--dark);
+}
+
+.appstore-btn:hover, .playstore-btn:hover {
+    background: var(--container);
+}
+
+.images {
+    display: flex;
+    gap: 40px;
+}
+
+.column1, .column2 {
+    display: grid;
+    gap: 40px;
+}
+
+.about-picture {
+    border-radius: 20px;
+}
+
+.about-picture.picture-1 {
+    width: 660px;
+    height: 590px;
+}
+
+.about-picture.picture-2 {
+    width: 660px;
+    height: 430px;
+}
+
+.about-picture.picture-3 {
```

Comment:
There might be a way to define the picture sizes using grid, but we can leave it for the next part of the project.